### PR TITLE
Create Issue on Deploy Failure

### PR DESCRIPTION
**Pull request by nickumia-reisys**
Related to
- https://github.com/GSA/data.gov/issues/2827

Depends on
- #574 

Important Notes:
- This creates an issue in this repo, but I think that's still okay.

References:
- https://github.com/nickumia-reisys/github-action-test/issues/5
- https://github.com/nickumia-reisys/github-action-test/pull/6
- https://docs.github.com/en/actions/learn-github-actions/contexts
- https://github.com/GSA/datagov-brokerpak-eks/blob/main/.github/workflows/test.yml
- https://github.com/JasonEtco/create-an-issue
- https://github.com/marketplace/actions/create-an-issue

### CODE REVIEW

File: .github/deploy-failure.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+title: Deployment Failure - {{ date | date('dddd, MMMM Do') }}
```
Comment (nickumia-reisys) [inline .github/deploy-failure.md:2]:
```suggestion
title: Deployment Failure
```

**Comment by FuhuXia** [comment]
TODO future: we want to see the issue created in the main repo.

**Review by FuhuXia — DISMISSED** [review]

**Comment by nickumia-reisys** [comment]
It's using the [official usage](https://github.com/JasonEtco/create-an-issue#inputs) (I've also [tested it](https://github.com/nickumia-reisys/github-action-test/blob/main/.github/workflows/issue_creation.yml#L34-L36) with this syntax and [it works](https://github.com/nickumia-reisys/github-action-test/actions/runs/3211272696)), but yaml isn't happy 😕 


image

It the second to last run, but the last run had the same issue. 😢 

@robert-bryson @FuhuXia Can I get some help here?

**Comment by robert-bryson** [comment]
> It's using the [official usage](https://github.com/JasonEtco/create-an-issue#inputs) (I've also [tested it](https://github.com/nickumia-reisys/github-action-test/blob/main/.github/workflows/issue_creation.yml#L34-L36) with this syntax and [it works](https://github.com/nickumia-reisys/github-action-test/actions/runs/3211272696)), but yaml isn't happy 😕
> 
> image
> 
> It the second to last run, but the last run had the same issue. 😢
> 
> @robert-bryson @FuhuXia Can I get some help here?

It looks like the emoji is working, right? You might also be able to do something like `"some text :cry:"`?

**Comment by nickumia-reisys** [comment]
Yeah, the emoji was the problem.  Sorry, @jbrown-xentity and I paired a bit and fixed it and I forgot to update the issue.  Thanks for looking at it though @robert-bryson 😄

**Review by btylerburton — APPROVED** [review]

**Pull request merged**